docs/_sass/_docs-display-component.scss

```diff
@@ -19,7 +19,7 @@
     }
```

Comment:
why does PR also have CSS changes? can they be handled separately?

Reply:
It needs the CSS changes per "I've removed any references to variables (besides colors) that were present in the docs site scss files (mostly fd-type('x'))"

It won't build if those aren't removed.